Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -172,13 +193,15 @@ class PushNotificationIOS {
       listener =  PushNotificationEmitter.addListener(
         DEVICE_NOTIF_EVENT,
         (notifData) => {
+          notifData.isRemote = true;
```

Comment:
This is now set on the native side: https://github.com/facebook/react-native/blob/master/Libraries/PushNotificationIOS/RCTPushNotificationManager.m#L162